test/tables.txt

```diff
@@ -488,20 +502,40 @@ aaa|bbb
 <th>aaa</th>
 </tr>
 </thead></table>
-````````````````````````````````
```

Comment:
A few of the previous tests were wrong when comparing against GitHub, and that got broken with the new whitespace handling, so I ended up replacing them